charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         image: {{ default .Values.global.image.repository .Values.repoServer.image.repository }}:{{ default .Values.global.image.tag .Values.repoServer.image.tag }}
         imagePullPolicy: {{ default .Values.global.image.imagePullPolicy .Values.repoServer.image.imagePullPolicy }}
         command:
-        - uid_entrypoint.sh
+        - entrypoint.sh
```

Comment:
maybe it is more sustainable to don't define this as command, but as args, so that the default entrypoint of the image is used? https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/

Reply:
@davidkarlsen I've updated the PR per your suggestion